src/@carbon-info/views/Data/Data.tsx

```diff
@@ -1,217 +1,243 @@
-    borderRadius: "4px",
-    margin: "auto",
-  },
-  cosmosSDK: {
-    background: "linear-gradient(180deg, #469590 0%, #31D8D6 100%)",
-    backgroundClip: "text",
-    WebkitTextFillColor: "transparent",
-    WebkitBackgroundClip: "text",
-  },
-  poweredBy: {
-    display: "flex",
-    justifyContent: "center",
-    alignItems: "center",
-    marginBottom: "6rem",
-    [theme.breakpoints.only("sm")]: {
-      marginBottom: "5rem",
-    },
-    [theme.breakpoints.only("xs")]: {
-      marginBottom: "4rem",
-    },
-  },
-  cosmosSDKLogo: {
-    margin: theme.spacing(0, 1.5, 0),
-    [theme.breakpoints.down("sm")]: {
-      margin: theme.spacing(-1, 1, 0),
-      transform: "scale(0.75)",
-    },
-  },
-  smallerText: {
-    [theme.breakpoints.down(380)]: {
-      fontSize: "1.45rem",
-    },
-  },
+	container: {
+		width: "100%",
+		display: "flex",
+		flexDirection: "column",
+		justifyContent: "center",
+		alignItems: "center",
+		position: "relative",
+		marginTop: "10rem",
+		paddingBottom: "5rem",
+		zIndex: 5,
+	},
+	headerText: {
+		maxWidth: "57.125rem",
+		textAlign: "center",
+		opacity: 0,
+		transition: "all 2s ease",
+		"&.open": {
+			opacity: 1,
+		},
+		[theme.breakpoints.down("sm")]: {
+			...theme.typography.h3,
+		},
+	},
+	dataWrapper: {
+		width: "100%",
+		left: 0,
+		[theme.breakpoints.up("xl")]: {
+			maxWidth: "1900px",
+		},
+	},
+	dataTable: {
+		marginTop: "8rem",
+		borderWidth: "2px",
+		marginLeft: "auto",
+		marginRight: "auto",
+		borderStyle: "solid",
+		borderColor: theme.palette.primary.main,
+		boxShadow: theme.shadows[3],
+		background:
+			"radial-gradient(63.65% 55% at 50.51% 100%, rgba(10, 220, 182, 0.25) 0%, rgba(10, 220, 182, 0.06) 51.54%, rgba(10, 220, 182, 0.00) 100%), #0F4950",
+		borderRadius: "20px",
+		opacity: 0,
+		transition: "all 2s ease",
+		animation: "expandTable .25s",
+		"&.open": {
+			opacity: 1,
+		},
+		"& > div": {
+			// Apply borderLeft to all child divs except the first one
+			"&:not(:first-child) > div": {
+				borderLeft: `2px solid ${theme.palette.primary.main}`,
+				[theme.breakpoints.down("xs")]: {
+					borderLeft: "0",
+				},
+			},
+
+			// Apply marginLeft and marginRight auto to all child divs except the first and last one
+			"& > div": {
+				marginLeft: "auto",
+				marginRight: "auto",
+			},
+		},
+	},
+	bannerContainer: {
+		marginTop: "4rem",
+		[theme.breakpoints.up("sm")]: {
+			marginTop: 0,
+			display: "flex",
+			justifyContent: "center",
+			position: "relative",
+			top: "-20px",
+		},
+	},
+	dataBox: {
+		margin: theme.spacing(4, 0),
+		borderRadius: 0,
+		backgroundColor: "transparent",
+		[theme.breakpoints.only("xs")]: {
+			margin: theme.spacing(2, 0),
+		},
+	},
+	mobileDivider: {
+		width: "20px",
+		height: "2px",
+		backgroundColor: theme.palette.background.divider,
+		boxShadow: theme.shadows[4],
+		borderRadius: "4px",
+		margin: "auto",
+	},
+	cosmosSDK: {
+		background: "linear-gradient(180deg, #469590 0%, #31D8D6 100%)",
```

Comment:
This CSS style is hardcoded under `StyleUtils` alr (found under `src/@carbon-info/utils/styles.ts`). 

Pls import from `src/@carbon-info/utils/styles.ts` and apply (the one you're looking for `StyleUtils.blueGradient`).